Frontend.py

```diff
@@ -1,60 +1,78 @@
 import os
+import backend
+
 bold_text = '\033[1m'
 end_color = '\033[0m'
 purple_text = '\033[95m'
 underline_text = '\033[4m'
 blue_text = '\033[96m'
 underline_text = '\033[4m'
-red_text = '\033[91m'
+#Idea - Text editor in the command line 
+#Does anyone have any other ideas 
+#C - add, view, main, stylstic choices (color, bold), help, introduction
+#T - search, markdown to PDF (I just figure we can add this feature because it shouldn't be too difficult), 
+#J - clear notes, delete files, list notes, edit notes (This one might be difficult to do)
 
 def Help(): 
     print(f"\n{bold_text}{purple_text}Noteworthy Help Manuel{end_color}\n")
     print(f"{blue_text}")
-    print("Type 'syntax' for information relating to Markdown Syntax")
-    print("Type 'date' for information about date formatting for this app\n")
     
-    help_variable = input("What do you need help with? ")
+    print(f"{underline_text}Help Functions{end_color}{blue_text}")
+    print("     syntax: information relating to Markdown Syntax")
+    print("     date: information about date formatting for this app")
+    print("     end_of_file: information about how to stop taking notes\n")
+    
+    help_variable = input(f"{underline_text}What do you need help with?{end_color}{blue_text} ")
 
     if help_variable == "syntax":
         print ("Markdown Syntax Guide: https://www.markdownguide.org/")
     elif help_variable == "date": 
-        print ("Date Format: mm-dd-yyyy")
+        print ("Date Format: yyyy-mm-dd")
+    elif help_variable == "end_of_file":
+        print ("Stop Writing to File: 'ctrl + D' ")
     else: 
         print ("Function not Available")
         exit()
+        
     
     print(f"{end_color}")
+    main()
     
 def AddNotes(): 
     print (f"{bold_text}{purple_text}\nAdd Notes{end_color}")
     print (f"{blue_text}")
-    name_of_file = input("Name of Markdown File to add to: ")
+    name_of_file = input(f"{underline_text}Name of Markdown File to add to:{end_color}{blue_text} ")
     name_of_file = f"Notes/{name_of_file}"
-    date_of_file = input(f"Input today's Date: ")
-
+    date_of_file = input(f"{underline_text}Input today's Date:{end_color}{blue_text} ")
+    date_of_file = f"Date: {date_of_file}"
     if not ".md" in name_of_file:
         name_of_file = f"{name_of_file}.md"
     
-    with open(name_of_file, 'a') as w: 
-        w.write(date_of_file)
-        print("Type notes here (Press return for new line and type END whenever you are done with your notes): ")
-        continue_adding = True
-        while continue_adding == True: 
-            notes = input()
-            if notes == "END": 
-                break
-            w.write('\n')
-            w.write(notes)
+    print(f"{underline_text}Type notes below: {end_color}{blue_text}\n")
+    print("     return key: Start a new line")
```

Comment:
I like that you added this! It really helps users who are used to return ending input know that we support multiline